src/content/members/members/derekjj.ts

```diff
@@ -2,7 +2,7 @@ import type { MemberObject } from '../types';
 // import { profileMasks } from '../flare';
 
 // Change _EXAMPLE to `yourGitHubUserName` and add your info below
-export const _EXAMPLE: MemberObject = {
```

Comment:
I found this error after committing the same error myself.  It showed up in the logs.

Reply:
**praise:** Good catch! Thanks, Ross!